R/message_generators.R

```diff
@@ -357,11 +372,14 @@ wrong_value <- function(this,
 
   glue::glue_data(
     list(
-      this = this,
+      intro = intro,
+      expected = expected,
       that = that,
+      where = if (!identical(this, "")) where else "",
+      this = this,
       action = action %||% ""
     ),
-    "{intro}I expected {action}{that} where you wrote {this}."
+    "{intro}I {expected} {action}{that}{where}{this}."
```

Comment:
😂

Reply:
I suppose I could move the _I_ and the white space into `expected` and make it

```
"{intro}{expected}{action}{that}{where}{this}"
```

😆